### DISCUSSION TITLE
Building a Conference Solution: Mastering the New Architecture on Ant Media

### DISCUSSION THREAD

**Discussion by zerostack0** [ant-media/Ant-Media-Server#5938] [Announcements]
🌟 Exciting Update: How to Create a Conference Solution with the new conference architecture on Top Ant Media? Join Our Learning Session! 🚀

Hello Everyone! 👋

We are absolutely thrilled to extend an invitation to you for an immersive session dedicated to exploring Ant Media's new conference architecture. This is a fantastic opportunity to gain a deep understanding of our latest features and developments.

Key Details:

    📅 Date: December 22nd
    ⏰ Time: 3:00 PM GMT+3
    🌐 Location: https://meet.antmedia.io/Conference/learningsession

🔗 Save the Date: Don't miss out! [Add to your calendar](https://calendar.google.com/calendar/u/0/r/eventedit/N2d2bmlhNXNrajZldDZ2NHJpcDhmMWdqaHFfMjAyMzEyMDFUMTEwMDAwWiBtdXN0YWZhLmJvbGVrZW5AYW50bWVkaWEuaW8)

📢 Reminder: We're planning weekly Learning Sessions. Be sure to keep an eye on your calendars!

Looking forward to seeing you there! 💻

**Comment by burak-58** [comment]
Hi All,
You can find the related improvements about the new architecture #5960 .
